test/test_connect.f90

```diff
@@ -1,11 +1,11 @@
-program name
-  use redis_mod
-  implicit none
+PROGRAM name
```

Comment:
Do you have a strong preference for upper-case statements? I usually just use lower-case everything in modern fortran code, as recommended [here](http://www.fortran90.org/src/best-practices.html). I just checked and WRF uses upper-case, but SAM and many other codes use lower-case.